src/fivetran-destination/src/destination/ddl.rs

```diff
@@ -49,29 +53,35 @@ pub async fn handle_describe_table(
     };
 
     let columns = {
+        let stmt = format!(
+            r#"SELECT
+                   name,
+                   type_oid,
+                   type_mod,
+                   CASE WHEN coms.comment = {magic_comment} THEN True ELSE false END AS primary_key
+               FROM mz_columns AS cols
+               JOIN mz_internal.mz_comments AS coms
+               ON cols.id = coms.id AND cols.position = coms.object_sub_id
+               WHERE cols.id = 'u1'"#,
```

Comment:
```suggestion
               WHERE cols.id = $2"#;
```

Reply:
Whoops, this was left over from testing, fixed!